### PR TITLE
Add geotic benchmark

### PR DESCRIPTION
**Pull request by ooflorent**
I had high expectation regarding Geotic considering its presentation, so I decided to add it to the suite.

> Geotic is an ECS library focused on **performance**, **features**, and **non-intrusive design**. Geotic is consistantly one of the fastest js ecs libraries, especially when it comes to large numbers of entities.

@ddmills: could you consider reviewing this? Maybe I did something wrong…

### CODE REVIEW

File: src/cases/geotic.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  let animates = world.createQuery({
+    all: [Animation],
+  });
+
+  let renderables = world.createQuery({
+    all: [Position, Render],
+  });
+
+  insertEntities(world, count);
```
Comment (ddmills) [inline src/cases/geotic.js:97]: 🔍 this can be moved to before the queries are created, to match other suites

**Review by ddmills — APPROVED** [review]
I think this is accurate usage of geotic! It really shines when components are created/destroyed over time, and components are added/removed while there are queries that are tracking them

File: src/cases/geotic.js (hunk 1 above)
Comment (ooflorent) [reply src/cases/geotic.js]: Done!

**Comment by ooflorent** [comment]
> It really shines when components are created/destroyed over time
That's good to hear! I'll add a third test using mutations.

**Pull request merged**